### PR TITLE
Update install.md

### PR DESCRIPTION
**Pull request by olgahryniuk**
Deleting ''Update the local project file to use the VRF library' instructions as suggested in this [thread](https://input-output-rnd.slack.com/archives/GR599HMFX/p1621237157125000)

### CODE REVIEW

File: doc/getting-started/install.md [hunk 1]
```diff
@@ -139,13 +139,7 @@ Checkout the latest version of cardano-node (choose the tag with the highest ver
 
 We explicitly use the GHC version that we installed earlier.  This avoids defaulting to a system version of GHC that might be older than the one you have installed.
 
-    cabal configure --with-compiler=ghc-8.10.2
-
-Update the local project file to use the VRF library that you installed earlier.
-
-    echo "package cardano-crypto-praos" >>  cabal.project.local
-    echo "  flags: -external-libsodium-vrf" >>  cabal.project.local
-
```
Comment (coot) [inline doc/getting-started/install.md:148]: It's better if we preserve this part, but add a comment that this is only for a development build.

btw, I don't know yet how to make it work without it.

**Review by coot — CHANGES_REQUESTED** [review]

File: doc/getting-started/install.md (hunk 1 above)
Comment (olgahryniuk) [reply doc/getting-started/install.md]: Thanks, a note is now added and that part is back in.

File: doc/getting-started/install.md [hunk 2]
```diff
@@ -139,14 +139,13 @@ Checkout the latest version of cardano-node (choose the tag with the highest ver
 
 We explicitly use the GHC version that we installed earlier.  This avoids defaulting to a system version of GHC that might be older than the one you have installed.
 
-    cabal configure --with-compiler=ghc-8.10.2
+    cabal configure --with-compiler=ghc-8.10.4
 
-Update the local project file to use the VRF library that you installed earlier.
+Note, that for a development build you can update the local project file to use the VRF library that you installed earlier:
```
Comment (coot) [inline doc/getting-started/install.md:144]:
```suggestion
Note, that for a development build you can avoid installing the custom `libsodium` library and add the following lines to the local project file:
```

**Review by coot — CHANGES_REQUESTED** [review]
Please squash the commits, and it's good to be merged.

**Review by coot — APPROVED** [review]

**Pull request merged**